doc/source/theory/dem/dem.rst

```diff
@@ -121,80 +124,122 @@ Where:
 
 * :math:`\nu_i, \nu_j` poisson coefficient of particle i or j;
 
-Rolling friction may be calculated through a constant torque model or a viscous torque model corresponding to those equations:
+~~~~~~~~~~~~~~~~~~~~
+Coulomb's limit
+~~~~~~~~~~~~~~~~~~~~
+
+Coulomb's criterion is breached when the following condition is broken during a collision:
 
 .. math::
-    \mathbf{M}_{ij}^{r} &= -\mu_{r}R_{e}|\mathbf{F}_{ij}^{n}| \mathbf{\hat{\omega}}_{ij} \\
-    \mathbf{M}_{ij}^{r} &= -\mu_{r}R_{e}|\mathbf{F}_{ij}^{n}||\mathbf{V}_{\omega}| \mathbf{\hat{\omega}}_{ij}
+    |\mathbf{F}_{ij}^{t}| \geq \mu |\mathbf{F}_{ij}^{n}|
 
-Where the parameters are:
+
+A breach means the collision is having gross sliding and tangential force needs to be limited to the Coulomb's limit.
```

Comment:
Yes, but what does it mean? Your version is only saying what is happening algorithmically, not physically...

Also, like I said in a previous comment to Bruno, my next PR is about the Coulomb limit, so I wont focus on modifying this in the documentation since it will change in the next two weeks.